glad/stages/change_pixel_depth.py

```diff
@@ -29,6 +31,9 @@ def change_pixel_depth(tiles, **kwargs):
                 "TILED=YES",
                 "-co",
                 "SPARSE_OK=TRUE",
+                "-tr",
+                PIXEL_WIDTH,
+                PIXEL_WIDTH,
```

Comment:
you also want to resampling method and override the output extent
`-r nearest`
`-a_ullr {min_x} {max_y} {max_x} {min_y}`

you can get the bbox based on the tile_id

```python
from utils import get_bbox_by_tile_id
min_x, max_y, max_x, min_y = get_bbox_by_tile_id(tile_id)
```

Reply:
@thomas-maschler it looks gdal_translate only lets you either use -tr OR -a_ullr + -outsize (otherwise throws an error). But -a_ullr and -outsize don't actually resample raster, they just change the geoference metadata. I noticed now though that -tr does create misshaped tiles (e.g. 40,000 x 40,251 tiles). 

So just changing to gdalwarp which will actually resample the tile when you set the output extent and height/width